lib/datapacktypes/omniscript.js

```diff
@@ -90,14 +91,28 @@ OmniScript.prototype.afterActivationSuccess = async function(inputMap) {
     var jobInfo = inputMap.jobInfo;
     var dataPack = inputMap.dataPack;
 
+    if(!jobInfo.checkLWCActivation){
```

Comment:
Should only check `if (!jobInfo.ignoreLWCActivation && !jobInfo.checkLWCActivation)`